src/OrchardCore.Modules/Orchard.Setup/Services/SetupContext.cs

```diff
@@ -12,7 +13,7 @@ public class SetupContext
         public string DatabaseProvider { get; set; }
         public string DatabaseConnectionString { get; set; }
         public string DatabaseTablePrefix { get; set; }
-        public IDictionary<string, string> RecipeExtraParams { get; set; }
+        public IDictionary<string, StringValues> RecipeExtraParams { get; set; }
```

Comment:
Request.Form stores data on StringValues that is why I replaced string per StringValues for not losing input data.